### ISSUE TITLE
Training strategy for Image Classification

### ISSUE THREAD

**Issue by williamberrios** [liaopeiyuan/artbench#4]
Would you mind sharing your training approach on this dataset (scheduler, hyperameters, etc) ?  Did you also get around 66% of acc on validation set with a resnet50?

**Comment by liaopeiyuan** [comment]
You may wanna refer to the default training method for https://github.com/rwightman/pytorch-image-models with SEResNet-34 Resolution 224 (then replace 224 with 256).
